Dockerfile

```diff
@@ -1,12 +1,8 @@
-FROM alpine:3.13.5
+FROM ubuntu:22.04
```

Comment:
Info: It seems latest alpine can not run eventrouter. I think if we build it from alpine then we can run it. I assume it is related to `alpine images are based on musl instead of glibc` issue. Not sure but i do not see any problem running it in ubuntu.

Reply:
We need to use ubuntu 20.04 because of the docker engine version used on central cluster.